tests/test_fork.py

```
@@ -460,3 +462,152 @@ def test_get_investment_universe_on_fork(sepolia_fork):
+            "from": from_address,
+            "to": to,
+            "data": data,
+            "gas": 3_000_000,
+        }
+    )
+    receipt = w3.eth.wait_for_transaction_receipt(tx_hash)
+    if receipt["status"] != 1:
+        raise AssertionError(
+            f"Impersonated tx from {from_address} to {to} failed "
+            f"(status={receipt['status']})"
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)